### ISSUE TITLE
Is there any plan to support echo cancllation?

### ISSUE THREAD

**Issue by yangwuan55** [google/oboe#85]
Just like the ``` AcousticEchoCanceler ``` from java can use ``` AudioTrack ``` and ``` AudioRecord ```.

**Comment by dturner** [comment]
@philburk Will the P features allow this? i.e. https://github.com/google/oboe/pull/99/commits/c30bbe603c256f92cdf2876c3122bc5be24b5e3e#diff-c178d9e54b1979b39c97a2d942d1c272

**Comment by philburk** [comment]
Yes, the P features will allow the use of effects such as AEC with AAudio and, therefore, with Oboe.

The effects will need to be controlled from Java using the existing AudioEffect API.

**Comment by strentler** [comment]
How is the current status?
Is there an example which shows how to enable AEC with Oboe?

**Comment by feliwir** [comment]
I'm interested in a sample of this for C++ aswell. **AcousticEchoCanceler** seems to be a pure Java/Kotlin library
